standards/code-style.adoc

```diff
@@ -253,15 +260,14 @@ non-static variables may occur.
 
 Example:
 
-[source, java]
+[source, kotlin]
 ----
-public class MyClass {
-    public static final int SOME_CONSTANT = 42;
-    private static float sValue; // Contrived but you get the point.
-    private int value;
+class MyClass {
+    val value = 0
 
-    public int getValue() {
-        return value;
+    companion object {
+        const val SOME_CONSTANT = 42
+        private const val sValue = 0f // Contrived but you get the point.
```

Comment:
Sorry to be pedantic, but `sValue` should not be `const` since it's demonstrating the equivalent of `static`, not `static final`.

Reply:
Updated